sdk/storage/inc/common/storage_url_builder.hpp

```diff
@@ -62,10 +56,9 @@ namespace Azure { namespace Storage {
       }
     }
 
-    void RemoveQuery(const std::string& key)
-    {
-      m_query.erase(key);
-    }
+    void RemoveQuery(const std::string& key) { m_query.erase(key); }
```

Comment:
>void RemoveQuery(const std::string& key) { m_query.erase(key); } [](start = 4, length = 64)

Jason used to insist that this should be split into multiple lines. Please ask Jason on this approach to confirm that he's also OK with this.

Reply:
This is formatted by .clang-format. I think it's ok.

if you don't think so maybe we should bring this up in weekly meetings.